src/core/schema/components/populate.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://udes.io./jsonschema/workflows/components/populate.json",
+    "title": "display_data",
+    "description": "Workflows populate attribute for component",
+    "type": "object",
+    "definitions": {
+        "validators": {
+            "type": "array",
+            "items": [{ "type": "string" }]
+        },
+        "path": { "type": "string" },
```

Comment:
this is probably overkill as there not particularly complex nor are they likely to become so

Reply:
simplified :)